components/onbee_app/actions/update-employee/update-employee.mjs

```diff
@@ -0,0 +1,66 @@
+import app from "../../onbee_app.app.mjs";
+
+export default {
+  key: "onbee_app-update-employee",
+  name: "Update Employee",
+  description: "Update an employee with the specified ID. [See the documentation](https://docs.onboardee.io/api/#tag/Employees/paths/~1employees~1edit~1{id}/post)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    app,
+    employeeId: {
+      propDefinition: [
+        app,
+        "employeeId",
+      ],
+    },
+    firstname: {
+      propDefinition: [
+        app,
+        "firstname",
+      ],
+    },
+    surname: {
+      propDefinition: [
+        app,
+        "surname",
+      ],
+    },

```

Comment:
The [docs](https://docs.onboardee.io/api/#tag/Employees/paths/~1employees~1edit~1{id}/post) show `firstname` and `surname` as being optional for editing an employee.
```suggestion
    firstname: {
      propDefinition: [
        app,
        "firstname",
      ],
      optional: true,
    },
    surname: {
      propDefinition: [
        app,
        "surname",
      ],
      optional: true,
    },
```